INSTALL.rst

```diff
@@ -458,12 +458,9 @@ they are not in the environment ``$PATH`` already.
 
 .. code-block:: console
 
-    # Global installation
-    $ sudo su -c "npm install -g less clean-css requirejs uglify-js"
-
-    # or
-    # Local installation
-    (invenio)$ npm install less clean-css requirejs uglify-js
+    # Local installation (using package.json)
+    (invenio)$ cdvirtualenv src/invenio
+    (invenio)$ npm install
```

Comment:
I don't think ``-g`` is supported from a ``package.json`` ? At least it did not work for me and could not find online how to do it.

http://stackoverflow.com/questions/6480549/install-dependencies-globally-and-locally-using-package-json